Cargo.toml

```diff
@@ -12,28 +12,35 @@ description = "Fuel cryptographic primitives."
 
 [dependencies]
 borrown = "0.1"
+coins-bip32 = { version = "0.7.0", default-features = false, optional = true }
```

Comment:
Last nit; use major/minor, unless we need to target a specific patch (in that case, we need to add a comment saying why)

Reply:
Oh yeah, overlooked this one. Should definitely not pin minor haha.